rule/add-constant.go

```diff
@@ -158,6 +214,18 @@ func (r *AddConstantRule) configure(arguments lint.Arguments) {
 						panic(fmt.Sprintf("Invalid argument to the add-constant rule, expecting string representation of an integer. Got '%v'", v))
 					}
 					r.strLitLimit = limit
+				case "ignoreFuncs":
+					excludes, ok := v.(string)
+					if !ok {
+						panic(fmt.Sprintf("Invalid argument to the add-constant rule, string expected. Got '%v' (%T)", v, v))
+					}
+
+					for _, exclude := range strings.Split(excludes, ",") {
```

Comment:
trimming blanks can help in avoiding subtle config problems.
For example configuring the rule with 
```go
"os.*,.Println,ignoredFunc"
```
will not behave as if it is configured with
```go
"os.*, .Println,ignoredFunc"
```